### PR TITLE
upcoming: [M3-8406, M3-8407] - Image Service Gen2 - Fix Image Capability and other tweaks

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Replaces incorrect `distributed-images` capability with `distributed-sites` ✏️ 
- Adds polling every 30 seconds to the Images landing table because we don't have any account events we can use to update Region statuses when replication finishes 🔄
- Enforces that least one "available" region is submitted in the Manage Regions Drawer 🟢 

## Target release date 🗓️
8/5

### Capability now shows in `dev`
Screenshot 2024-07-30 at 3 44 38 PM

### Enforcing at least one "Available" Region

Screenshot 2024-07-31 at 1 09 07 PM

Screenshot 2024-07-31 at 1 08 59 PM

### Prerequisites
- You need the `image-gen2` account tag 🏷️ 

### Verification steps
- Test the `distributed-sites` capability in `dev` by uploading an Image
  - Verify the new image has the capability once uploaded
- Test the validation / disabled button logic in the "Manage Regions Drawer"
  - I recommend testing this in `staging` by uploading an Image

## As an Author I have considered 🤔

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Comment by hkhalil-akamai** [comment]
I'm not sure if this would ever be a valid state, but I can't add any regions to an image that currently is not replicated anywhere:

https://github.com/user-attachments/assets/945a2c5c-2a93-451a-9406-c57591905b63

**Review by hkhalil-akamai — APPROVED** [review]
Changes look good but noticing one issue. Approving to expedite for this release.

**Comment by bnussman-akamai** [comment]
@hkhalil-akamai  Good catch. I pushed a change that hides the "Manage Regions" action menu option if the Image doesn't have `regions`.  6bf6648

File: packages/manager/src/features/Images/ImagesLanding/ImageRegions/ImageRegionRow.tsx [hunk 1]
```diff
@@ -39,14 +40,24 @@ export const ImageRegionRow = (props: Props) => {
...
         />
-        <IconButton
-          aria-label={`Remove ${region}`}
-          disabled={disableRemoveButton}
-          onClick={onRemove}
-          sx={{ p: 0.5 }}
+        <Tooltip
+          title={
+            disableRemoveButton
+              ? 'You cannot remove this region because at least one available region must be present.'
```
Comment (carrillo-erik) [inline packages/manager/src/features/Images/ImagesLanding/ImageRegions/ImageRegionRow.tsx:46]: I noticed this string is used in 2 other files, given that we've had to update text copy several times in other instances, I would opt for creating a `constants.ts` file when possible in the future.

**Review by jaalah-akamai — APPROVED** [review]

**Pull request merged**